tests/test_config.py

```diff
@@ -19,6 +19,8 @@ def conf_path():
 
 def test_config_creation(conf_path):
     conf = config.read_config(file_path=conf_path)
+    print(conf_path)
+    print(conf)
```

Comment:
are these prints still necessary?

Reply:
No, removed in last push! I'll merge then